### ISSUE TITLE
Simplify dates

### ISSUE THREAD

**Issue by jeroen** [jeroen/mongolite#81]
The problem is that `jsonlite` uses `unlist()` to simplify, which unclasses the datetime object:

```
library(mongolite)
m <- mongo()
m$insert('{"mydate":[{"$date":"2016-10-11T00:00:00Z"},{"$date":"2016-10-12T00:00:00Z"}]}')
x <- m$find()
x$mydate
```

**Comment by jeroen** [comment]
This should be fixed by updating to jsonlite 1.4:

```r
devtools::install_github("jeroen/jsonlite")
```

**Comment by jeroen** [comment]
There is now also `mongo_options(date_as_char = TRUE)`
